src/polyfills/intersection-observer.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * See https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver.
+ */
+
+import {IntersectionObserverStub} from '../polyfillstub/intersection-observer-stub';
+
+/**
+ * @param {!Window} win
+ */
+export function install(win) {
```

Comment:
I think the word "install" is used to mean "stub", "fetch" and "upgrade" in different places which is tough for readability. Can we call this `stub`?

Reply:
I renamed all others to `upgrade` per your suggestions above. But I'd rather keep calling this one `install()` b/c I think this hides the fact of stubbing from most of places as an implementation choice. But I also redid the "fetch" part as well to reduce number of entities. PTAL.